<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#282a36">
<meta name="generator" content="Hexo 6.3.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/tritiumlogo-260x260.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/tritiumlogo-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/tritiumlogo-16x16.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#282a36">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"tritium.cloud","root":"/","scheme":"Muse","version":"7.8.0","exturl":false,"sidebar":{"position":"right","display":"hide","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta name="description" content="Hey you, you&#39;re finally awake">
<meta property="og:type" content="website">
<meta property="og:title" content="TRITIUM CLOUD 氚">
<meta property="og:url" content="http://tritium.cloud/page/3/index.html">
<meta property="og:site_name" content="TRITIUM CLOUD 氚">
<meta property="og:description" content="Hey you, you&#39;re finally awake">
<meta property="og:locale" content="en_US">
<meta property="article:author" content="Ao Zhang">
<meta property="article:tag" content="Computer Science, Machine Learning">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="http://tritium.cloud/page/3/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : true,
    isPost : false,
    lang   : 'en'
  };
</script>

  <title>TRITIUM CLOUD 氚</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

  <script src="https://fastly.jsdelivr.net/gh/Billyzhang1229/hexo-live2d-widget@v1.0.5/autoload.js"></script>
<!-- hexo injector head_end start -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/hexo-math@4.0.0/dist/style.css">
<!-- hexo injector head_end end --></head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">TRITIUM CLOUD 氚</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">By Ao Zhang</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>Archives</a>

  </li>
        <li class="menu-item menu-item-beta">

    <a href="https://beta.tritium.cloud/" rel="noopener" target="_blank"><i class="fa fa-umbrella-beach fa-fw"></i>Beta</a>

  </li>
        <li class="menu-item menu-item-genomics">

    <a href="/tags/genomics/" rel="section"><i class="fa fa-dna fa-fw"></i>Genomics</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <div class="reading-progress-bar"></div>

  <a href="https://github.com/Billyzhang1229" class="github-corner" title="Follow me on GitHub" aria-label="Follow me on GitHub" rel="noopener" target="_blank"><svg width="80" height="80" viewBox="0 0 250 250" aria-hidden="true"><path d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"></path><path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path><path d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z" fill="currentColor" class="octo-body"></path></svg></a>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content index posts-expand">
            
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="en">
    <link itemprop="mainEntityOfPage" href="http://tritium.cloud/2023/11/03/Principals-Underlying-Basic-Molecular-Genetic-Techniques/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="Ao Zhang">
      <meta itemprop="description" content="Hey you, you're finally awake">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="TRITIUM CLOUD 氚">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2023/11/03/Principals-Underlying-Basic-Molecular-Genetic-Techniques/" class="post-title-link" itemprop="url">Principals Underlying Basic Molecular Genetic Techniques</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">Posted on</span>

              <time title="Created: 2023-11-03 19:21:30" itemprop="dateCreated datePublished" datetime="2023-11-03T19:21:30Z">2023-11-03</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">Edited on</span>
                <time title="Modified: 2023-11-05 17:54:42" itemprop="dateModified" datetime="2023-11-05T17:54:42Z">2023-11-05</time>
              </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="principals-underlying-basic-molecular-genetic-techniques"><a class="markdownIt-Anchor" href="#principals-underlying-basic-molecular-genetic-techniques"></a> Principals Underlying Basic Molecular Genetic Techniques</h1>
<p>Key Technologies to Study and Analyze Genes</p>
<h2 id="dna-cloning"><a class="markdownIt-Anchor" href="#dna-cloning"></a> DNA Cloning</h2>
<ul>
<li>purify and amplify DNA fragments of interest</li>
<li>recombinant DNA technology</li>
<li><strong>Recombinant DNA</strong>: DNA that has been created artificially by combining DNA from different sources</li>
</ul>
<hr />
<ul>
<li>DNA fragments lack of capability of initiating replication</li>
<li><strong>Replicon</strong>: Replication origin in a molecules</li>
<li>Human DNA fragments need a suitable replicon to replicate within a bacterial cell.</li>
<li>The result is the formation of a recombinant DNA.</li>
</ul>
<h3 id="insertion-of-dna-fragments-into-cloning-vectors"><a class="markdownIt-Anchor" href="#insertion-of-dna-fragments-into-cloning-vectors"></a> Insertion of DNA Fragments into Cloning Vectors</h3>
<h3 id="overview"><a class="markdownIt-Anchor" href="#overview"></a> Overview</h3>
<ul>
<li><strong>Isolated human DNA is long</strong> and needs to be fragmented for cloning.</li>
<li>Cloning vectors can only accept relatively <strong>small DNA molecules</strong>.</li>
<li>Fragmentation by physical shearing is not sufficient as it produces <strong>long fragments with heterogeneous ends</strong>.</li>
<li>To facilitate ligation, <strong>small DNA fragments with uniform ends</strong> are required.</li>
</ul>
<h3 id="tools-required"><a class="markdownIt-Anchor" href="#tools-required"></a> Tools Required</h3>
<ul>
<li><strong>Restriction Enzymes</strong>: Bacterial enzymes that recognize and cleave specific DNA sequences.</li>
<li><strong>DNA Ligases</strong>: Enzymes that <strong>join DNA fragments</strong> together.</li>
</ul>
<h3 id="restriction-endonucleases"><a class="markdownIt-Anchor" href="#restriction-endonucleases"></a> Restriction Endonucleases</h3>
<ul>
<li>They recognize specific <strong>short sequence elements</strong> within double-stranded DNA.</li>
<li>Cleavage occurs on both strands within or close to these sequences.</li>
<li>Bacteria produce <strong>restriction endonucleases with different sequence specificities</strong>.</li>
</ul>
<h3 id="restriction-sites"><a class="markdownIt-Anchor" href="#restriction-sites"></a> Restriction Sites</h3>
<ul>
<li>Often <strong>short palindromic sequences</strong>.</li>
<li>Read the same in the <strong>5ʹ to 3ʹ direction</strong> on both DNA strands.</li>
<li>Cleavage may produce <strong>overhanging 5ʹ or 3ʹ ends</strong> (sticky ends) or <strong>blunt ends</strong>.</li>
</ul>
<h3 id="cloning-vectors"><a class="markdownIt-Anchor" href="#cloning-vectors"></a> Cloning Vectors</h3>
<ul>
<li>DNA fragments with <strong>sticky or blunt ends</strong> can be inserted into vectors.</li>
<li>Vectors are <strong>engineered to contain unique restriction sites</strong> for specific enzymes.<br />
Sure! I’ll use an analogy that draws a parallel between recombinant DNA technology and concepts familiar to computer scientists:</li>
</ul>
<hr />
<h3 id="plasmid-vectors"><a class="markdownIt-Anchor" href="#plasmid-vectors"></a> Plasmid Vectors</h3>
<ul>
<li>Think of <strong>plasmids</strong> as “external hard drives” that are not part of the computer’s main storage (cell’s chromosomal DNA).</li>
<li>Just like you would use a specific file format (e.g., FAT32, NTFS) for a hard drive to be readable by most PCs, <strong>E. coli</strong> plasmids are like the most commonly used in DNA technology.</li>
<li>Each “external hard drive” (plasmid) should have:
<ul>
<li>A <strong>boot sector (ORI)</strong> to initiate its operation.</li>
<li><strong>File encryption or password protection</strong> (selectable marker) to prevent unwanted access (here, it’s usually a drug-resistance gene to ward off antibiotics).</li>
<li>A <strong>folder or directory</strong> (region) where new files (<strong>exogenous DNA fragments</strong>) can be added.</li>
</ul>
</li>
</ul>
<hr />
<h3 id="recombinant-dna-process"><a class="markdownIt-Anchor" href="#recombinant-dna-process"></a> Recombinant DNA Process</h3>
<ol>
<li><strong>Copy-pasting files</strong>: Combine your “external hard drive” (<strong>plasmid vector</strong>) with the file (<strong>DNA fragment</strong>) you want to add.</li>
<li>Using a special software tool, akin to a <strong>file compressor or zipping tool</strong>, you insert the file into the external hard drive, resulting in an updated version of the hard drive (<strong>recombinant plasmid</strong>).</li>
<li>To ensure the file gets written correctly, you connect the hard drive to a computer (<strong>E. coli</strong>). This process might require certain conditions, like a specific cable or software (here, it’s <strong>CaCl₂</strong> and a heat-pulse).</li>
<li><strong>Checking for errors</strong>: After writing, you test if the file was copied correctly by running a specific application. In biology, this is like culturing cells on a plate with <strong>ampicillin</strong>.</li>
<li>If the file (DNA fragment) was copied correctly to the external hard drive (plasmid), the system (<strong>transformed cell</strong>) will operate without errors. If not, the system crashes or doesn’t run (cells that didn’t take up the plasmid die).</li>
<li><strong>Auto-backup</strong>: Once the file is safely stored, the system automatically creates multiple backups (<strong>plasmid replication</strong>).</li>
<li><strong>End result</strong>: You have many systems or computers (<strong>colony of cells</strong>) each having their own copy of the external hard drive with the added file (recombinant plasmid).</li>
</ol>
<h3 id="dna-library"><a class="markdownIt-Anchor" href="#dna-library"></a> DNA Library</h3>
<ul>
<li>
<p><strong>Genomic library</strong>:</p>
<ul>
<li>DNA library: DNA molecules are cloned into a vector molecule.</li>
<li>Genomic library: a collection of DNA fragments that represent the entire genome of an organism.</li>
</ul>
</li>
<li>
<p><strong>Complementary DNA (cDNA) library</strong>:</p>
<ul>
<li>cDNA library: represents the mRNA expressed in a cell type.</li>
<li>only expressed genes are represented.</li>
</ul>
</li>
</ul>
<h3 id="dna-cloning-limitations"><a class="markdownIt-Anchor" href="#dna-cloning-limitations"></a> DNA Cloning Limitations</h3>
<ul>
<li>
<p><strong>Laborious and time-consuming</strong></p>
</li>
<li>
<p><strong>Not suitable for parallel amplification of many DNA fragments from diferent sources</strong></p>
</li>
</ul>
<h2 id="pcr-polymerase-chain-reaction"><a class="markdownIt-Anchor" href="#pcr-polymerase-chain-reaction"></a> PCR (Polymerase Chain Reaction)</h2>
<p>PCR is a technique that allows the amplification of a specific DNA fragment with gene-specific primers.</p>
<p>It is a <strong>cell-free</strong> method</p>
<p>Pros:</p>
<ul>
<li><strong>Fast</strong> (minimum 1 hour)</li>
<li><strong>Parallel amplification</strong> of many DNA fragments from different sources</li>
<li><strong>Highly sensitive</strong> (can detect a single DNA molecule)</li>
</ul>
<p>The Polymerase Chain Reaction (PCR) is a technique used in molecular biology to <strong>amplify</strong> a segment of DNA. Below is the summary in key steps:</p>
<h3 id="pcr-steps"><a class="markdownIt-Anchor" href="#pcr-steps"></a> PCR Steps</h3>
<p><img src="https://upload.wikimedia.org/wikipedia/commons/a/ab/Polymerase_chain_reaction-en.svg" alt="Polymerase chain reaction" /></p>
<blockquote>
<p><em>Enzoklop, CC BY-SA 4.0 <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-sa/4.0">https://creativecommons.org/licenses/by-sa/4.0</a>, via Wikimedia Commons</em></p>
</blockquote>
<ol>
<li>
<p><strong>Preparation</strong>: Gather necessary components:</p>
<ul>
<li><strong>DNA template</strong> with the target sequence.</li>
<li><strong>Heat-stable DNA polymerase</strong> enzyme.</li>
<li><strong>Two PCR primers</strong> for the start and end of the target sequence.</li>
<li><strong>Deoxynucleotide triphosphates (dNTPs)</strong>, the building blocks for new DNA.</li>
<li><strong>Reaction buffer</strong> with <strong>Magnesium ions</strong> for enzyme activity.</li>
<li>A <strong>thermal cycler</strong> for temperature control.</li>
</ul>
</li>
<li>
<p><strong>Denaturation</strong>: Heat the mixture to make the DNA <strong>single-stranded</strong>.</p>
</li>
<li>
<p><strong>Annealing</strong>: Cool down so primers can <strong>bind</strong> to their complementary DNA sequences.</p>
</li>
<li>
<p><strong>Extension</strong>: Raise temperature for DNA polymerase to <strong>synthesize new DNA strands</strong>.</p>
</li>
<li>
<p><strong>Amplification</strong>: Repeat the cycle multiple times, <strong>doubling</strong> the DNA amount each cycle.</p>
</li>
<li>
<p><strong>Finalization</strong>: End with billions of copies of the specific DNA segment.</p>
</li>
</ol>
<p>This process allows for the DNA to be <strong>sufficiently amplified</strong> for further analysis.</p>
<h3 id="pcr-cycle"><a class="markdownIt-Anchor" href="#pcr-cycle"></a> PCR Cycle</h3>
<ul>
<li><strong>Denaturation</strong></li>
<li><strong>Annealing</strong></li>
<li><strong>Extension</strong></li>
</ul>
<p>This <strong>Cycle</strong> is repeated <strong>25-35</strong> times</p>
<h3 id="pcr-primers"><a class="markdownIt-Anchor" href="#pcr-primers"></a> PCR Primers</h3>
<p><strong>Designing PCR Primers</strong></p>
<ul>
<li>
<p><strong>Specificity</strong>:</p>
<ul>
<li><strong>Primer sequence</strong>: Primers should only bind to the target sequence.ce:</li>
<li><strong>Length</strong>: 17-28bp optimal.</li>
<li><strong>Nature of the template</strong>:
<ul>
<li><strong>Melting temperature (Tm)</strong>:
<ul>
<li>Optimal Tm: 52-65°C</li>
<li>Influenced by GC content, length, and concentration</li>
<li>Anneling temperature: 5°C below Tm</li>
<li>Working Approximation: <strong>Tm = 4(G+C) + 2(A+T)</strong></li>
</ul>
</li>
<li><strong>GC content</strong>: 45-55% optimal</li>
</ul>
</li>
</ul>
</li>
<li>
<p><strong>Secondary structures and primers homology</strong></p>
<ul>
<li>Avoid regions of self-homology (<strong>within the sequence</strong>)</li>
</ul>
<blockquote>
<p><strong>Self-homology</strong>: the presence of complementary regions within a single nucleic acid molecule.</p>
</blockquote>
<ul>
<li>Avoid inter-primer homology that could lead to primer dimers (<strong>between two primers</strong>)</li>
</ul>
<blockquote>
<p><strong>inter-primer homology</strong>: the presence of complementary regions between two different primers.</p>
</blockquote>
</li>
<li>
<p><strong>Amplicon size</strong></p>
<ul>
<li>Avoid large amplicon size</li>
</ul>
</li>
<li>
<p><strong>Repeats and Runs</strong></p>
<ul>
<li>Long runs of <strong>single base</strong> repeats larger than <strong>4bp</strong> should be avoided</li>
<li>Long runs of <strong>di-nucleotide repeats</strong> should be avoided</li>
</ul>
</li>
</ul>
<h2 id="gene-expression-analysis-techniques"><a class="markdownIt-Anchor" href="#gene-expression-analysis-techniques"></a> Gene Expression Analysis Techniques</h2>
<h3 id="endpoint-of-pcr"><a class="markdownIt-Anchor" href="#endpoint-of-pcr"></a> Endpoint of PCR</h3>
<p>Measures the amount of DNA at the end of the reaction, rather than during each cycle. It is semi-quantitive which means that it can be used to compare the amount of DNA in different samples, but it cannot be used to determine the exact amount of DNA in a sample.</p>
<ul>
<li><strong>Easy and Cheap</strong></li>
<li><strong>Not very accurate</strong></li>
</ul>
<p><strong>Angarose gel electrophoresis</strong> is used to separate DNA fragments by size. The DNA is stained with a dye that fluoresces under UV light. The DNA is visualized as bands on the gel.</p>
<p><img src="/images/genomics/Gel-Electrophoresis.webp" alt="Angarose gel electrophoresis" /></p>
<blockquote>
<p><em>Genome Research Limited | License: CC BY 4.0</em></p>
</blockquote>
<h3 id="real-time-pcr-qpcr"><a class="markdownIt-Anchor" href="#real-time-pcr-qpcr"></a> Real-time PCR (qPCR)</h3>
<p>Measures the amount of DNA during each cycle of the reaction. Exponecial phase of the reaction is the best time to collect and analyse data.</p>
<h3 id="sybr-green-i-and-taqman-assays"><a class="markdownIt-Anchor" href="#sybr-green-i-and-taqman-assays"></a> SYBR Green I and TaqMan assays</h3>
<p><img src="/images/genomics/SYBR_green_TaqMan_assay.png" alt="SYBR Green I and TaqMan assays" /></p>
<blockquote>
<p><em>Cao, Yiqi &amp; Luo, Yu-Feng et al. (2020). Digital PCR as an Emerging Tool for Monitoring of Microbial Biodegradation. Molecules. 25. 706. 10.3390/molecules25030706.</em></p>
</blockquote>
<h3 id="taqman-probe-based"><a class="markdownIt-Anchor" href="#taqman-probe-based"></a> TaqMan Probe-Based</h3>
<ul>
<li><strong>TaqMan probe</strong>: a short single-stranded DNA molecule that is complementary to the target sequence.</li>
</ul>
<h3 id="hybridization-assay"><a class="markdownIt-Anchor" href="#hybridization-assay"></a> Hybridization assay</h3>
<ul>
<li><strong>Hybridization</strong>: the process of forming a double-stranded nucleic acid molecule from two complementary single strands of DNA or RNA.</li>
</ul>
<h2 id="why-do-we-measure-gene-expression"><a class="markdownIt-Anchor" href="#why-do-we-measure-gene-expression"></a> Why do we measure gene expression?</h2>
<ul>
<li>gene expression is identical across all cells</li>
<li>tissue-specific gene expression</li>
<li>disease-specific gene expression</li>
<li>gene expression changes over time</li>
</ul>
<h3 id="methods-for-gene-transcription-profiling"><a class="markdownIt-Anchor" href="#methods-for-gene-transcription-profiling"></a> Methods for gene transcription profiling</h3>
<ul>
<li><strong>Microarray</strong></li>
<li><strong>RNA-seq</strong></li>
<li><strong>RT-PCR</strong></li>
</ul>
<h3 id="rt-pcr-reverse-transcription-pcr"><a class="markdownIt-Anchor" href="#rt-pcr-reverse-transcription-pcr"></a> RT-PCR (Reverse Transcription PCR)</h3>
<ul>
<li><strong>Reverse transcription</strong>: the process of synthesizing a DNA molecule from an RNA template.</li>
</ul>
<p><strong>Why?</strong>:</p>
<ul>
<li>RNA is unstable and easily degraded</li>
<li>In order to have more RNA for analysis, we need to convert it to a more stable form (complementary DNA, cDNA)</li>
</ul>
<p><strong>Components</strong>:</p>
<ul>
<li><strong>cDNA</strong>: complementary DNA</li>
<li><strong>DNA polymerase</strong>: enzyme that synthesizes DNA</li>
<li><strong>Two primers</strong>: one for the start and one for the end of the target sequence</li>
<li><strong>dNTPs</strong>: building blocks for new DNA</li>
<li>Reaction buffer with <strong>Magnesium ions</strong> for enzyme activity</li>
<li>Thermal cycler for temperature control</li>
</ul>
<p><strong>Negative control</strong>:</p>
<ul>
<li>No reverse transcriptase</li>
<li>No template</li>
</ul>
<h2 id="why-examine-protein-expression"><a class="markdownIt-Anchor" href="#why-examine-protein-expression"></a> Why examine protein expression?</h2>
<ul>
<li>RNA is not always translated into protein</li>
<li>Study correlation between RNA and protein expression</li>
<li>Examine post-translational modifications</li>
</ul>
<h3 id="quantification-of-protein-expression"><a class="markdownIt-Anchor" href="#quantification-of-protein-expression"></a> Quantification of protein expression</h3>
<ul>
<li>Enzyme-linked immunosorbent assay (ELISA)</li>
<li>Western blotting</li>
<li>Immunohistochemistry (IHC)</li>
</ul>
<blockquote>
<p><strong>Semi-quantitative methods</strong>: indirect measurement of expression<br />
<strong>Quantitative methods</strong>: direct measurement of expression</p>
</blockquote>
<p><strong>Western blotting</strong> is a semi-quantitative method that uses antibodies to detect specific proteins in a sample.</p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  


  
  <nav class="pagination">
    <a class="extend prev" rel="prev" href="/page/2/"><i class="fa fa-angle-left" aria-label="Previous page"></i></a><a class="page-number" href="/">1</a><a class="page-number" href="/page/2/">2</a><span class="page-number current">3</span><a class="page-number" href="/page/4/">4</a><span class="space">&hellip;</span><a class="page-number" href="/page/11/">11</a><a class="extend next" rel="next" href="/page/4/"><i class="fa fa-angle-right" aria-label="Next page"></i></a>
  </nav>



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">
      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">Ao Zhang</p>
  <div class="site-description" itemprop="description">Hey you, you're finally awake</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">11</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
      <div class="site-state-item site-state-tags">
        <span class="site-state-item-count">32</span>
        <span class="site-state-item-name">tags</span>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="https://github.com/billyzhang1229" title="GitHub → https:&#x2F;&#x2F;github.com&#x2F;billyzhang1229" rel="noopener" target="_blank"><i class="fab fa-github fa-fw"></i>GitHub</a>
      </span>
      <span class="links-of-author-item">
        <a href="mailto:ao.zhang@kcl.ac.uk" title="E-Mail → mailto:ao.zhang@kcl.ac.uk" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
  </div>



      </div>
      <script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=a&t=n&d=rZBHlWQ6Lzt3Tj6s5c_b7muHM7QOI9RB2Bf8-u5SWxs&co=222222&cmo=d8b647&cmn=35dd6c&ct=808080'></script>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2023</span>
  <span class="with-love">
    <i class="fas fa-terminal"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Ao Zhang</span>
</div>
  <div class="powered-by">Powered by <a href="https://hexo.io/" class="theme-link" rel="noopener" target="_blank">Hexo</a> & <a href="https://muse.theme-next.org/" class="theme-link" rel="noopener" target="_blank">NexT.Muse</a>
  </div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/pjax/pjax.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>

  <script>
var pjax = new Pjax({
  selectors: [
    'head title',
    '#page-configurations',
    '.content-wrap',
    '.post-toc-wrap',
    '.languages',
    '#pjax'
  ],
  switches: {
    '.post-toc-wrap': Pjax.switches.innerHTML
  },
  analytics: false,
  cacheBust: false,
  scrollTo : !CONFIG.bookmark.enable
});

window.addEventListener('pjax:success', () => {
  document.querySelectorAll('script[data-pjax], script#page-configurations, #pjax script').forEach(element => {
    var code = element.text || element.textContent || element.innerHTML || '';
    var parent = element.parentNode;
    parent.removeChild(element);
    var script = document.createElement('script');
    if (element.id) {
      script.id = element.id;
    }
    if (element.className) {
      script.className = element.className;
    }
    if (element.type) {
      script.type = element.type;
    }
    if (element.src) {
      script.src = element.src;
      // Force synchronous loading of peripheral JS.
      script.async = false;
    }
    if (element.dataset.pjax !== undefined) {
      script.dataset.pjax = '';
    }
    if (code !== '') {
      script.appendChild(document.createTextNode(code));
    }
    parent.appendChild(script);
  });
  NexT.boot.refresh();
  // Define Motion Sequence & Bootstrap Motion.
  if (CONFIG.motion.enable) {
    NexT.motion.integrator
      .init()
      .add(NexT.motion.middleWares.subMenu)
      .add(NexT.motion.middleWares.postList)
      .bootstrap();
  }
  NexT.utils.updateSidebarPosition();
});
</script>




  















    <div id="pjax">
  

  
      

<script>
  if (typeof MathJax === 'undefined') {
    window.MathJax = {
      loader: {
          load: ['[tex]/mhchem'],
        source: {
          '[tex]/amsCd': '[tex]/amscd',
          '[tex]/AMScd': '[tex]/amscd'
        }
      },
      tex: {
        inlineMath: {'[+]': [['$', '$']]},
          packages: {'[+]': ['mhchem']},
        tags: 'ams'
      },
      options: {
        renderActions: {
          findScript: [10, doc => {
            document.querySelectorAll('script[type^="math/tex"]').forEach(node => {
              const display = !!node.type.match(/; *mode=display/);
              const math = new doc.options.MathItem(node.textContent, doc.inputJax[0], display);
              const text = document.createTextNode('');
              node.parentNode.replaceChild(text, node);
              math.start = {node: text, delim: '', n: 0};
              math.end = {node: text, delim: '', n: 0};
              doc.math.push(math);
            });
          }, '', false],
          insertedScript: [200, () => {
            document.querySelectorAll('mjx-container').forEach(node => {
              let target = node.parentNode;
              if (target.nodeName.toLowerCase() === 'li') {
                target.parentNode.classList.add('has-jax');
              }
            });
          }, '', false]
        }
      }
    };
    (function () {
      var script = document.createElement('script');
      script.src = '//cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js';
      script.defer = true;
      document.head.appendChild(script);
    })();
  } else {
    MathJax.startup.document.state(0);
    MathJax.texReset();
    MathJax.typeset();
  }
</script>

    

  

    </div>
</body>
</html>
